classes/Bullet.py

```diff
@@ -7,18 +8,18 @@ def __init__(self,position,groups):
 
         self.pos=pygame.math.Vector2(self.rect.topleft)
         self.direction=pygame.math.Vector2(0,-1) #-1 for upwards 0 for left/ right
-        self.speed=600
+        self.speed=1000
         
-    def hit_target(self,target_group,scoreCount):
+    def hit_target(self,target_group):
         if pygame.sprite.spritecollide(self,target_group,True):
-            scoreCount+=1
+            Score.varScore+=1
```

Comment:
This logic of incrementing the score rather than using the pygame library function to increment the score looks fine for me. Great work!!